packages/react/src/components/UIShell/SkipToContent.tsx

```diff
@@ -28,28 +30,21 @@ function SkipToContent({
 
 SkipToContent.propTypes = {
   /**
-   * Provide text to display in the SkipToContent `a` tag
+   * A ReactNode to display in the SkipToContent `a` tag.
+   * `'Skip to main content'` by default.
    */
-  children: PropTypes.string.isRequired,
```

Comment:
I think restricting this to a string was intentional

Reply:
What if someone wants to put their own custom button or other component there? I decided to stick with the type what React has for the `children` prop of an `a` rather than overriding it. Let me know if that makes sense, otherwise I can restrict it to a string

Reply:
I just didn't want to unnecessarily restrict people, but I can change it back and adjust the type so that it only accepts string children